Tools/AP_Bootloader/support.cpp

```diff
@@ -562,13 +567,15 @@ void check_ecc_errors(void)
         // DMA'ble memory not available
         return;
```

Comment:
we need to re-enable the fault IRQ here, or only disable after the malloc_dma() call

Reply:
Fixed